### DISCUSSION TITLE
Trouble testing out TUS Uploady: Refused to get unsafe header: "Location"

### DISCUSSION THREAD

**Discussion by tjv255** [rpldy/react-uploady#484] [General]
I am trying to give my component the ability to pause and resume an upload. I have been reading up on the TusUploady provider, which saves the file into local storage so it can be resumed later. 

Once switching providers from Uploady to TusUploady, I started getting the following error:
_createUpload.js:21` Refused to get unsafe header "Location"_

People online have said to add "Access-Control-Expose-Headers": "Location" to the request header, but that doesn't work. I believe this might be related to the TUS provider as the TUS documentation states that the response header must contain the URL of the created resource in the server (https://tus.io/protocols/resumable-upload.html).

For my server, am simply using a Webhooks.site as a mock backend. I've tried using RequestBin and Onboarding as alternatives with the same results. I also tried using the getMockSenderEnhancer which doesn't throw any errors, but also doesn't seem to support resuming cancelled tasks.

Has anyone dealt with this issue before with TusUploady? Any information will be appreciated. Thanks.

**Comment by yoavniran** [comment]
hi @tjv255 
To Test Tus, you do indeed need a server that follows the standard or you will not get real results.
The easiest way I found is to use one of the freely available implementation listed [here](https://tus.io/implementations.html).
There's a node server version and a tusd one that are very easy to use.
